charts/k8s-monitoring/charts/feature-pod-logs/values.yaml

```diff
@@ -127,15 +125,16 @@ labelsToKeep:
   - k8s.node.name
 
 # -- The structured metadata mappings to set.
-# To not set any structured metadata, set this to an empty object (e.g. `{}`)
 # Format: `<key>: <extracted_key>`.
 # Example:
 # structuredMetadata:
 #   component: component
 #   kind: kind
 #   name: name
 # @section -- Processing settings
-structuredMetadata: {}
+structuredMetadata:
+  k8s.pod.name: k8s.pod.name
+  pod: pod
```

Comment:
Is there a reason for using dots instead of underscores in the keys?

It's confusing that I have to override the default `labels` using underscores but I have to use dots for overriding the default `structuredMetadata`:

```yaml
podLogs:
  labels:
    app_kubernetes_io_name: ~
  structuredMetadata:
    k8s.pod.name: ~
```

When I change `k8s.pod.name:` to `k8s_pod_name:`, it won't override the default and the same mapping will be rendered twice in `stage.structured_metadata {...}`. Similarly, I cannot use dots in the `labels` configuration, because the helm chart doesn't escape the keys in `labels:`, but it will escape the keys in `structuredMetadata:`.